test/client/formatBidsAndAsks/index.ts

```diff
@@ -1,386 +1,386 @@
-import BigNumber from 'bignumber.js'
-import assert from 'assert-diff'
-import {Client} from 'xrpl-client'
-import requests from '../../fixtures/requests'
-import responses from '../../fixtures/responses'
+// import BigNumber from 'bignumber.js'
+// import assert from 'assert-diff'
+// import {Client} from 'xrpl-client'
+// import requests from '../../fixtures/requests'
+// import responses from '../../fixtures/responses'
 import {TestSuite} from '../../utils'
 
-function checkSortingOfOrders(orders) {
-  let previousRate = '0'
-  for (var i = 0; i < orders.length; i++) {
-    const order = orders[i]
-    let rate
+// function checkSortingOfOrders(orders) {
+//   let previousRate = '0'
+//   for (var i = 0; i < orders.length; i++) {
+//     const order = orders[i]
+//     let rate
 
-    // We calculate the quality of output/input here as a test.
-    // This won't hold in general because when output and input amounts get tiny,
-    // the quality can differ significantly. However, the offer stays in the
-    // order book where it was originally placed. It would be more consistent
-    // to check the quality from the offer book, but for the test data set,
-    // this calculation holds.
+//     // We calculate the quality of output/input here as a test.
+//     // This won't hold in general because when output and input amounts get tiny,
+//     // the quality can differ significantly. However, the offer stays in the
+//     // order book where it was originally placed. It would be more consistent
+//     // to check the quality from the offer book, but for the test data set,
+//     // this calculation holds.
 
-    if (order.specification.direction === 'buy') {
-      rate = new BigNumber(order.specification.quantity.value)
-        .dividedBy(order.specification.totalPrice.value)
-        .toString()
-    } else {
-      rate = new BigNumber(order.specification.totalPrice.value)
-        .dividedBy(order.specification.quantity.value)
-        .toString()
-    }
-    assert(
-      new BigNumber(rate).isGreaterThanOrEqualTo(previousRate),
-      'Rates must be sorted from least to greatest: ' +
-        rate +
-        ' should be >= ' +
-        previousRate
-    )
-    previousRate = rate
-  }
-  return true
-}
+//     if (order.specification.direction === 'buy') {
+//       rate = new BigNumber(order.specification.quantity.value)
+//         .dividedBy(order.specification.totalPrice.value)
+//         .toString()
+//     } else {
```

Comment:
Why'd we comment this one out?

Reply:
The fixtures are incorrectly formatted, and it seemed unnecessary to correct the fixtures when we're almost certainly getting rid of the method.